.github/workflows/push.yaml

```diff
@@ -10,7 +10,7 @@ on:
       - "server/**"
       - "terraform/**"
       - "Dockerfile"
-      - "!**/README.md"
```

Comment:
This is mean't to prevent builds/deploys when only a README file changes. We can leave that in place.